tests/core/test_loglevel.py

```diff
@@ -0,0 +1,72 @@
+import subprocess
+
+SCRIPT_CONTENT = """
+import logging as log
+def main(args, cijoe):
+    log.critical("critical")
+    log.error("error")
+    log.warning("warning")
+    log.info("info")
+    log.debug("debug")
+    return 0
+"""
```

Comment:
The tests here will probably only trigger the case of executing a script-directly, not when executing a workflow. Right? Perhaps that should covered as well?

Reply:
With log level being set in `parse_args()`, I don't think there would be a difference in calling **cijoe** with a script or with a workflow.